docker/judgehost/chroot-and-tar.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -eu
```

Comment:
```suggestion
#!/bin/bash -euo pipefail
```

Currently we don't have a loop but let's already protect ourselves for the future.

Reply:
Done (force-pushed).

Reply:
Oops, ShellCheck complains (correctly). I've switched to `set` (which is better anyway, because it also applies when the script is executed using `bash ./build.sh` rather than `./build.sh`).